docker/test/integration/MiNiFi_integration_test_driver.py

```diff
@@ -272,3 +272,11 @@ def wait_for_kafka_consumer_to_be_registered(self, cluster_name):
     def check_query_results(self, cluster_name, query, number_of_rows, timeout_seconds):
         cluster = self.acquire_cluster(cluster_name)
         assert cluster.check_query_results(query, number_of_rows, timeout_seconds)
+
+    def check_minifi_log_contents(self, line):
+        line_found = False
+        for cluster in self.clusters.values():
+            if cluster.get_engine() == "minifi-cpp":
+                line_found = cluster.wait_for_app_logs(line, 60)
+        assert line_found
+
```

Comment:
one of the builds fail as this contains an empty line at the end of file

Reply:
Thank-you for pointing it out. Fixed it :)